From 7fd9de7f12718db0ace575c15c3a0e061ef6c542 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Thu, 11 Nov 2021 09:46:41 +0100
Subject: [PATCH] m4/stack-direction.m4: fix build on microblazeel

Fix the following build failure on microblazeel with diffutils raised
since version 3.8 and
https://git.savannah.gnu.org/cgit/gnulib.git/commit/?id=e54aa6196947ed22ff66bcd714e4fc7bd0c5c3b4

sigsegv.c: In function 'sigsegv_handler':
sigsegv.c:979:28: error: 'struct vma_struct' has no member named 'is_near_this'
  979 |                       : vma.is_near_this (addr, &vma))
      |                            ^

Fixes:
 - http://autobuild.buildroot.org/results/5ecd55b7c72f9630a7acc72341ccb60e94ddb574

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: sent to bug-gnulib@gnu.org]
---
 m4/stack-direction.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/stack-direction.m4 b/m4/stack-direction.m4
index c90d69733..70286c8f5 100644
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -37,7 +37,7 @@ AC_DEFUN([SV_STACK_DIRECTION],
       m68* | \
       m88k | \
       mcore | \
-      microblaze | \
+      microblaze* | \
       mips* | \
       mmix | \
       mn10200 | \
-- 
2.33.0

